### PR TITLE
Search claims

### PR DESCRIPTION
**Pull request by falklinus**
## What?
- Search claim by id, number or member id

## Why?
-

## If applicable, rooms for improvement?
- Add to hunter and search type in regular search

## Optional checklist
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/portals/hope/features/claims/claims-list/LargeClaimsList.tsx [hunk 1]
```diff
@@ -110,6 +114,28 @@ export const LargeClaimsList: React.FC<{
...
+      filterMarkets: settings.marketFilterClaims,
+      filterTypesOfContract: hasCarPermission
+        ? TypeOfContractByInsuranceType.SWEDISH_CAR
+        : null,
+      filterClaimOutcomes: settings.outcomeFilterClaims,
+      searchString,
+    }
+
+    listClaims({
+      page: page - 1 ?? 0,
```
Comment (vonElfvin) [inline src/portals/hope/features/claims/claims-list/LargeClaimsList.tsx:134]: How would `page - 1` be `undefined` or `null`?  `undefined - 1` is `NaN`
```suggestion
      page: page - 1 ,
```

Comment (vonElfvin) [reply src/portals/hope/features/claims/claims-list/LargeClaimsList.tsx:134]: Does `??` work on `NaN`?

**Review by vonElfvin — APPROVED** [review]
Cool!

File: src/portals/hope/features/claims/claims-list/LargeClaimsList.tsx (hunk 1 above)
Comment (falklinus) [reply src/portals/hope/features/claims/claims-list/LargeClaimsList.tsx:134]: Good point. This was a lazy copy of the existing implementation. I'll correct it

**Pull request merged**